src/vc_util/src/lib.rs

```diff
@@ -20,9 +20,8 @@ use std::ops::{Add, Deref, DerefMut};
 
 pub mod issuer_api;
 
-pub const II_CREDENTIAL_URL_PREFIX: &str =
-    "https://internetcomputer.org/credential/internet-identity/";
-pub const II_ISSUER_URL: &str = "https://internetcomputer.org/issuers/internet-identity";
+pub const II_CREDENTIAL_URL_PREFIX: &str = "https://identity.internetcomputer.org/credential/";
+pub const II_ISSUER_URL: &str = "https://identity.internetcomputer.org/";
```

Comment:
Please use `https://identity.ic0.app`. The `https://internetcomputer.org` based one is not the recommended one (yet) since we have halted the domain migration.